consensus/src/main/java/org/apache/iotdb/consensus/ratis/ApplicationStateMachineProxy.java

```diff
@@ -132,7 +136,17 @@ public CompletableFuture<Message> applyTransaction(TransactionContext trx) {
         }
         IConsensusRequest deserializedRequest =
             applicationStateMachine.deserializeRequest(applicationRequest);
+
+        long startWriteTime = System.nanoTime();
         TSStatus result = applicationStateMachine.write(deserializedRequest);
+        MetricService.getInstance()
```

Comment:
we need to distinguish between local and remote apply to make sense of recording only the former